packages/service/src/utils/users.ts

```diff
@@ -1,5 +1,26 @@
+import { upsert } from "zapatos/db";
 import { db, pool, schema } from "../db";
 
 export function createUser(props: { name: string }) {
-  return db.upsert("users", props, "name").run(pool);
+  // return db.upsert("users", props, "name").run(pool);
+  return db.sql<
+    schema.users.SQL,
+    schema.users.Selectable
```

Comment:
I don't think this return type matches what the SQL is returing

Reply:
This type is still not correct. The variable name is misleading. This is a list still.

Reply:
maybe we could try out the `selectExactlyOne` helper